From: I Hsin Cheng <richard120310@gmail.com>
Subject: docs/mm: physical memory: Remove zone_t
Date: Wed, 15 Jan 2025 15:03:55 +0800

"zone_t" doesn't exist in current code base anymore, remove the
description of it.

Link: https://lkml.kernel.org/r/20250115070355.41769-1-richard120310@gmail.com
Signed-off-by: I Hsin Cheng <richard120310@gmail.com>
Reviewed-by: Randy Dunlap <rdunlap@infradead.org>
Cc: Ching-Chun (Jim) Huang <jserv@ccns.ncku.edu.tw>
Cc: Jonathan Corbet <corbet@lwn.net>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/mm/physical_memory.rst |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Documentation/mm/physical_memory.rst~docs-mm-physical-memory-remove-zone_t
+++ a/Documentation/mm/physical_memory.rst
@@ -33,7 +33,7 @@ The entire physical address space is par
 called zones which represent ranges within memory. These ranges are usually
 determined by architectural constraints for accessing the physical memory.
 The memory range within a node that corresponds to a particular zone is
-described by a ``struct zone``, typedeffed to ``zone_t``. Each zone has
+described by a ``struct zone``. Each zone has
 one of the types described below.
 
 * ``ZONE_DMA`` and ``ZONE_DMA32`` historically represented memory suitable for
_
